gschem/src/g_rc.c

```diff
@@ -128,8 +128,8 @@ SCM g_rc_gschem_version(SCM scm_version)
 SCM g_rc_net_direction_mode(SCM mode)
 {
   static const vstbl_entry mode_table[] = {
-    {TRUE , "enabled" },
-    {FALSE, "disabled"}
+    {TRUE , (char*) "enabled" },
+    {FALSE, (char*) "disabled"}
```

Comment:
As I mentioned elsewhere, usually it is best to fix these by changing `struct` definitions or function prototypes to use `const char*`.